src/Utopia/Messaging/Adapters/SMS/ClickSend.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+// Reference Material
+// https://developers.clicksend.com/docs/rest/v3/#send-sms
```

Comment:
Please move this to the top of the class declaration for consistency with other adapters

Reply:
Done